tst/EnergyPlus/unit/SetPointManager.unit.cc

```diff
@@ -1156,3 +1156,49 @@ TEST_F( EnergyPlusFixture, ColdestSetPointMgrInSingleDuct ) {
 
 }
 
+TEST_F( EnergyPlusFixture, SetPointManager_OutdoorAirResetMaxTempTest )
+{
+	bool ErrorsFound = false;
+
+	std::string const idf_objects = delimited_string( {
+		"Version,8.7;",
+
+		"  SetpointManager:OutdoorAirReset,",
+		"    Hot Water Loop Setpoint Manager,  !- Name",
+		"    MaximumTemperature,      !- Control Variable",
+		"    80.0,                    !- Setpoint at Outdoor Low Temperature {C}",
+		"    -17.778,                 !- Outdoor Low Temperature {C}",
+		"    40.0,                    !- Setpoint at Outdoor High Temperature {C}",
+		"    21.11,                   !- Outdoor High Temperature {C}",
+		"    HW Supply Outlet Node;   !- Setpoint Node or NodeList Name",
+
+	} );
+
+	ASSERT_FALSE( process_idf( idf_objects ) );
+	EXPECT_FALSE( ErrorsFound ); // zones are specified in the idf snippet
+
+	SetPointManager::GetSetPointManagerInputs();
+	// check Set Point Manager get inputs
+	EXPECT_EQ( SetPointManager::OutAirSetPtMgr( 1 ).CtrlVarType, "MAXIMUMTEMPERATURE" );
+	EXPECT_EQ( SetPointManager::OutAirSetPtMgr( 1 ).CtrlTypeMode, SetPointManager::iCtrlVarType_MaxTemp );
+	EXPECT_EQ( SetPointManager::AllSetPtMgr( 1 ).SPMType, SetPointManager::iSPMType_OutsideAir );
+	EXPECT_EQ( 80.0, SetPointManager::OutAirSetPtMgr( 1 ).OutLowSetPt1 );
+	EXPECT_EQ( -17.778, SetPointManager::OutAirSetPtMgr( 1 ).OutLow1 );
+	EXPECT_EQ( 40.0, SetPointManager::OutAirSetPtMgr( 1 ).OutHighSetPt1 );
+	EXPECT_EQ( 21.11, SetPointManager::OutAirSetPtMgr( 1 ).OutHigh1 );
+
+	SetPointManager::InitSetPointManagers();
```

Comment:
Should set the outdoor air temperature before this so that the setpoint manager does a real calculation.  The expects below will need to change now that the whole method has been changed.